sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs

```diff
@@ -86,35 +87,40 @@ internal static IReadOnlyList<FormContent> ConvertTextReferences(IReadOnlyList<s
             return formContent;
         }
 
+        private static Regex _wordRegex = new Regex(@"/readResults/(?<pageIndex>\d*)/lines/(?<lineIndex>\d*)/words/(?<wordIndex>\d*)");
+        private static Regex _lineRegex = new Regex(@"/readResults/(?<pageIndex>\d*)/lines/(?<lineIndex>\d*)");
```

Comment:
Good approach!

Reply:
waaa a comment in two lines! 😎

Reply:
I'm wondering if using Regex has unwanted performance implications, but it is tidy to write this way.  I'll ask around about perf concerns more tomorrow.

Reply:
Many thanks to @tg-msft for helping make these more performant and robust!